.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: Continous Integration
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9"] #, "3.10", "3.11", "3.12"]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install
+        run: pip install .[tests]
+      - name: Run tests
+        run: pytest --junitxml=junit/test-results.xml --cov=com --cov-report=xml --cov-report=html
```

Comment:
Is it useful to generate the coverage report if that is not used?

What about uploading the report HTML and maybe XML as a build artifact? That way it could be inspected at least.

Reply:
No its not. I just copied the command I ran locally, didn't figured out yet how to store the artifacts. I am going to dig into this now, thank you for the reminder!

Reply:
I think I configured it to post comments to the PRs, however this is hard to test :/

Reply:
I created a test case in my own repo by merging this and filing a new PR: https://github.com/georgkrause/pylistenbrainz/pull/2

Reply:
The display is a little bit confusing because tests are part of the package and the display doesn't add categories for the different python versions.

Reply:
I see. But I think that would be fine that's something that can be improved, and if something shows up there one can always start looking into the actual artifacts as well.

I assume the reason why the actions haven't been run here is because they need someone with write access (I don't) to trigger the run, right? Or maybe actions have to be enabled in the repo settings first.